_sass/_components/icon-list.scss

```diff
@@ -1,6 +1,6 @@
 .icon-list {
   display: flex;
-  margin-bottom: $paragraph-margins;
```

Comment:
We might consider keeping the `margin-bottom`. They list items seem a little close to me, especially on the homepage icon-list